pkg/codegen/templates/client.tmpl

```diff
@@ -1,14 +1,19 @@
 // RequestEditorFn  is the function signature for the RequestEditor callback function
 type RequestEditorFn func(req *http.Request, ctx context.Context) error
 
+// Doer performs HTTP requests
```

Comment:
I guess it would be nice to have an additional example comment and possibly an example where one mentions that `http.Client` implements this interface